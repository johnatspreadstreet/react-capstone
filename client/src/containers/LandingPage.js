import React, { Component } from "react";

export default class LandingPage extends Component {
  render() {
    return (
      <div>
        <h1>API Hub</h1>
        <p>A wonderful landing page will eventually go here</p>
        <button>Browse APIs</button>
      </div>
    );
  }
}
