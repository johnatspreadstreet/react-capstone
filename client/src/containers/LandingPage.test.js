import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from './LandingPage';

describe('A suite of tests for the App component', () => {
  // Smoke test
  it('should render with no errors', () => {
    const component = shallow(<LandingPage />);

    expect(component).toMatchSnapshot();
  });
});
