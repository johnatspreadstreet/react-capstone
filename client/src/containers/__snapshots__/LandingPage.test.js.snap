// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite of tests for the App component should render with no errors 1`] = `
<div>
  <h1>
    API Hub
  </h1>
  <p>
    A wonderful landing page will eventually go here
  </p>
  <button>
    Browse APIs
  </button>
</div>
`;
